src/seedu/addressbook/data/AddressBook.java

```diff
@@ -22,13 +24,15 @@
 
     private final UniquePersonList allPersons;
     private final UniqueTagList allTags; // can contain tags not attached to any person
-
+    private final Set<Tagging> taggings;// only record the tagging during this session
```

Comment:
good use of interface here so that it is easy to switch the implementation later